coq.dev opam package fails to build

<!-- Thank you for reporting a bug to Coq! -->

#### Description of the problem

<!-- If you can, it's helpful to provide self-contained example of some code
that reproduces the bug. If not, a link to a larger example is also helpful. -->
The coq.dev opam package fails to build, meaning many of our CI jobs are currently failing:

```
#=== ERROR while compiling coq.dev ============================================#
# context              2.0.3 | linux/x86_64 | ocaml-base-compiler.4.07.0 | pinned(git+https://github.com/coq/coq.git#master#924468eb)
# path                 /builds/iris/stdpp/opamroot/ocaml-base-compiler.4.07.0/.opam-switch/build/coq.dev
# command              /usr/bin/make -j39
# exit-code            2
# env-file             /builds/iris/stdpp/opamroot/log/coq-749-865888.env
# output-file          /builds/iris/stdpp/opamroot/log/coq-749-865888.out
### output ###
# [...]
# Fatal Error: Failure: Validation failed: tuple size: found 11, expected 12 (in mutual_inductive_body).
#              Please report at http://coq.inria.fr/bugs/.
# Makefile.build:826: recipe for target 'theories/Logic/Hurkens.vo' failed
# make[1]: *** [theories/Logic/Hurkens.vo] Error 1
# COQCHK    Coq.Numbers.DecimalFacts
# Fatal Error: Failure: Validation failed: tuple size: found 11, expected 12 (in mutual_inductive_body).
#              Please report at http://coq.inria.fr/bugs/.
# Makefile.build:826: recipe for target 'theories/Numbers/DecimalFacts.vo' failed
# make[1]: *** [theories/Numbers/DecimalFacts.vo] Error 1
# make[1]: Leaving directory '/builds/iris/stdpp/opamroot/ocaml-base-compiler.4.07.0/.opam-switch/build/coq.dev'
# Makefile:166: recipe for target 'submake' failed
# make: *** [submake] Error 2
```

Is building master broken, somehow?

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -170,7 +170,7 @@ after_script:
     - cd _install_ci
     - find lib/coq/ -name '*.vo' -print0 > vofiles
     - for regexp in 's/.vo//' 's:lib/coq/plugins:Coq:' 's:lib/coq/theories:Coq:' 's:/:.:g'; do sed -z -i "$regexp" vofiles; done
-    - xargs -0 --arg-file=vofiles bin/coqchk -boot -silent -o -m -coqlib lib/coq/
+    - xargs -0 --arg-file=vofiles bin/coqchk -silent -o -m -coqlib lib/coq/
 
 .ci-template: &ci-template
   stage: test

--- a/Makefile.build
+++ b/Makefile.build
@@ -550,11 +550,11 @@ $(CSDPCERTBYTE): $(CSDPCERTCMO)
 
 .PHONY: validate check test-suite $(ALLSTDLIB).v
 
-VALIDOPTS=$(if $(VERBOSE),,-silent) -o -m
+VALIDOPTS=$(if $(VERBOSE),,-silent) -o -m -coqlib .
 
 validate: $(CHICKEN) | $(ALLVO:.$(VO)=.vo)
 	$(SHOW)'COQCHK <theories & plugins>'
-	$(HIDE)$(CHICKEN) -boot $(VALIDOPTS) $(ALLMODS)
+	$(HIDE)$(CHICKEN) $(VALIDOPTS) $(ALLMODS)
 
 $(ALLSTDLIB).v:
 	$(SHOW)'MAKE $(notdir $@)'
@@ -828,7 +828,7 @@ theories/Init/%.vio: theories/Init/%.v $(VO_TOOLS_DEP)
 	$(HIDE)$(BOOTCOQC) $< $(TIMING_ARG) $(TIMING_EXTRA)
 ifdef VALIDATE
 	$(SHOW)'COQCHK    $(call vo_to_mod,$@)'
-	$(HIDE)$(CHICKEN) -boot -silent -norec $(call vo_to_mod,$@) \
+	$(HIDE)$(CHICKEN) $(VALIDOPTS) -norec $(call vo_to_mod,$@) \
 	  || ( RV=$$?; rm -f "$@"; exit $${RV} )
 endif
 

--- a/checker/checker.ml
+++ b/checker/checker.ml
@@ -189,6 +189,7 @@ let print_usage_channel co command =
 \n  -admit module          load module and dependencies without checking\
 \n  -norec module          check module but admit dependencies without checking\
 \n\
+\n  -coqlib dir            set coqchk's standard library location\
 \n  -where                 print coqchk's standard library location and exit\
 \n  -v                     print coqchk version and exit\
 \n  -boot                  boot mode\

